Orange/data/table.py

```diff
@@ -11,7 +11,6 @@
 from tempfile import NamedTemporaryFile
 from urllib.parse import urlparse, unquote as urlunquote
 from urllib.request import urlopen
-from urllib.error import URLError
```

Comment:
I think this might have been meant to be caught in the `from_url` method. :worried: